docs/rpc/http/getTokenAccountsByOwner.mdx

```diff
@@ -8,30 +8,30 @@ altRoutes:
 
 Returns all SPL Token accounts by token owner.
 
-<DocSideBySide>
+This method requires two parameters:
+
+1. The owner's public key
+2. An object specifying either the `mint` or the `programId`
```

Comment:
```suggestion
2. Either the `mint` or the `programId`
```

Reply:
Please don't close discussions i you don't want to implement them. Instead say why you don't like the suggestion.